DESCRIPTION

```diff
@@ -25,7 +25,8 @@ Imports:
     RcppParallel (>= 5.0.1),
     rlang,
     rstan (>= 2.18.1),
-    rstantools (>= 2.3.1.1)
+    rstantools (>= 2.3.1.1),
+    deSolve
```

Comment:
I would add this to suggest rather than imports. It's not essential for package to function

Reply:
Done.